redfish-core/lib/cable.hpp

```diff
@@ -60,17 +99,236 @@ inline void
             // Cable length is NaN by default, do not throw an error
             if (!std::isnan(*length))
             {
-                messages::internalError(resp);
+                messages::internalError(asyncResp->res);
                 return;
             }
         }
         else
         {
-            resp.jsonValue["LengthMeters"] = *length;
+            asyncResp->res.jsonValue["LengthMeters"] = *length;
         }
     }
 }
 
+/**
+ * @brief Fill cable specific properties.
+ * @param[in,out]   asyncResp       Async HTTP response.
+ * @param[in]       ec          Error code corresponding to Async method call.
```

Comment:
nit: ec and properties no longer parameters
 cableObjectPath and service new parameters

Reply:
Fixed